### PR TITLE
[Tests-only] Use ocis-phoenix for running tests in drone

### PR DESCRIPTION
**Pull request by dpakach**
## Description

In CI right now we use the ocis binary to run phoenix service for acceptance tests. This PR changes that to ocis-phoenix binary so that we can catch error even earlier.

## Related Issue

- Fixes <issue_link>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: .drone.star [hunk 1]
```diff
@@ -1046,8 +1047,7 @@ def ocisServices():
 		},
 		'commands': [
 			'cd /var/www/owncloud',
-			'./ocis devldap &',
```
Comment (kiranparajuli589) [inline .drone.star]: should that be deleted too?

File: .drone.star [hunk 2]
```diff
@@ -89,6 +89,7 @@ config = {
 			},
 			'runningOnOCIS': True,
 			'filterTags': 'not @skip and not @skipOnOCIS',
+			'screenShots': True,
```
Comment (individual-it) [inline .drone.star]: debug or do we need the screenshots for longterm?

File: .drone.star (hunk 1 above)
Comment (dpakach) [reply .drone.star]: yes, that should not have been there in the first place.

File: .drone.star (hunk 2 above)
Comment (dpakach) [reply .drone.star]: I think we can leave it for now, will be helpful as soon as tests fail in ocis.

**Pull request merged**